controllers/front/return.php

```diff
@@ -55,6 +112,7 @@ public function postProcess()
 
```

Comment:
Account To Account payment method can return status pending. Merchants with this payment method could have bussiness licence. In this case you are skipping this logic for this case